pkg/core/storage/redis_store_test.go

```diff
@@ -0,0 +1,182 @@
+package storage
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/CityOfZion/neo-go/config"
+	"github.com/alicebob/miniredis"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewRedisBatch(t *testing.T) {
+	want := &RedisBatch{mem: map[string]string{}}
+	if got := NewRedisBatch(); !reflect.DeepEqual(got, want) {
+		t.Errorf("NewRedisBatch() = %v, want %v", got, want)
+	}
+}
+
+func TestNewRedisStore(t *testing.T) {
+	redisMock, redisStore := prepareRedisMock(t)
+	key := []byte("foo")
+	value := []byte("bar")
+	err := redisStore.Put(key, value)
```

Comment:
I'm not sure about this test, you already set this key-value (foo-bar) pair in the `prepareRedisMock()`, so maybe it's better to use something else here?